packages/amplify-category-custom/resources/cdk-stack.ts.sample

```diff
@@ -1,10 +1,10 @@
-import * as cdk from '@aws-cdk/core';
+import * as cdk from 'aws-cdk-lib';
 import * as AmplifyHelpers from '@aws-amplify/cli-extensibility-helper';
 import { AmplifyDependentResourcesAttributes } from '../../types/amplify-dependent-resources-ref';
-//import * as iam from '@aws-cdk/aws-iam';
-//import * as sns from '@aws-cdk/aws-sns';
-//import * as subs from '@aws-cdk/aws-sns-subscriptions';
-//import * as sqs from '@aws-cdk/aws-sqs';
+//import * as iam from 'aws-cdk-lib/aws-iam';
+//import * as sns from 'aws-cdk-lib/aws-sns';
+//import * as subs from 'aws-cdk-lib/aws-sns-subscriptions';
+//import * as sqs from 'aws-cdk-lib/aws-sqs';
```

Comment:
Do these imports still make sense to include now that there's only 1 cdk package?

Reply:
yes , we still need to import these to use them in code. But there is only one package to install as dependency i.e. ```aws-cdk-lib``` instead of ```@aws/cdk/aws-iam , @aws-cdk/aws-sns...``` and so on.